priv/repo/migrations/20230803115906_add_columns_in_organization.exs

```diff
@@ -4,7 +4,7 @@ defmodule Glific.Repo.Migrations.AddSettingColumnInOrganization do
   def up do
     alter table(:organizations) do
       add(:parent_org, :varchar)
-      add(:setting, :jsonb)
+      add(:setting, :jsonb, default: "{}")
```

Comment:
setting column is already there in organization table

Reply:
Alright, Created a new migration which just modifies the default value of setting to be {}